sig/handwritten/app/jobs/application_job.rbs

```diff
@@ -0,0 +1,3 @@
+class ApplicationJob < ActiveJob::Base
+  include _RbsRailsPathHelpers
```

Comment:
I added this `include` statement to use path helper methods in the Job subclasses.